back/engines/commercial/content_builder/spec/services/content_builder/layout_image_service_spec.rb

```diff
@@ -241,5 +276,58 @@
       output = service.render_data_images imageable.craftjs_json
       expect(output).to eq expected_json
     end
+
+    it 'can deal with craftjs_json in an unexpected format' do
```

Comment:
What's unexpected about this format? Maybe a comment would help to explain?

Reply:
@jinjagit I just added two comments to elaborate :)